js/views/adScreen.jsx

```diff
@@ -15,6 +15,8 @@ import Utils from '../components/utils';
  * The screen to be displayed on advertising
  */
 class AdScreen extends React.Component {
+  isMounted = false;
```

Comment:
This is obsolete. Component will not receive new prop if it's unmounted

Reply:
also replace `componentWillReceiveProps` with `componentDidUpdate`

Reply:
Thank you Alexey! @ooyala-amohnatkin